src/index.js

```diff
@@ -1,3 +1,137 @@
+// 전역 변수
+const todoItem = `
+<div class="view">
+  <input class="toggle" type="checkbox"/>
+  <label class="label">새로운 타이틀</label>
+  <button class="destroy"></button>
+</div>
+<input class="edit" value="새로운 타이틀" />
+`;
```

Comment:
```suggestion
const todoItemTemplate = `
<div class="view">
  <input class="toggle" type="checkbox"/>
  <label class="label">새로운 타이틀</label>
  <button class="destroy"></button>
</div>
<input class="edit" value="새로운 타이틀" />
`;
```
보통 todoItem 이라는 변수명만 보면 템플릿이라고 유추하기가 쉽지않습니다. 명시적으로 template을 붙여주는게 어떨까요?

Reply:
좋은 의견 감사드립니다!
확실히 알아보기 쉬워진 것같아요